src/CreateZip.cpp

```diff
@@ -543,29 +543,25 @@ ulg crc32(ulg crc, const uch *buf, size_t len)
 class TZip
 {
 public:
-	TZip() : pfout(nullptr),ooffset(0),oerr(false),writ(0),hasputcen(false),zfis(0),hfin(0)
-	{
-	}
-	~TZip()
-	{
-	}
+	TZip() = default;
+	~TZip() = default;
 
 	// These variables say about the file we're writing into
 	// We can write to pipe, file-by-handle, file-by-name, memory-to-memmapfile
-	RageFile *pfout;             // if valid, we'll write here (for files or pipes)
-	unsigned ooffset;         // for pfout, this is where the pointer was initially
-	ZRESULT oerr;             // did a write operation give rise to an error?
-	unsigned writ;            // how have we written. This is maintained by Add, not write(), to avoid confusion over seeks
-	unsigned int opos;        // current pos in the mmap
-	unsigned int mapsize;     // the size of the map we created
-	bool hasputcen;           // have we yet placed the central directory?
+	RageFile *pfout{nullptr};             // if valid, we'll write here (for files or pipes)

```

Comment:
I've put the code back to the way it was.

FWIW, I prefer initializing member variables where they are declared because
- You don't have to remember to initialize the variable in the constructor.
  - It's also easier to tell if a variable was initialized, especially in classes with several member variables.
- You don't have to duplicate initialization in multiple constructors.
- When using constructors, variables need to be initialized in the order they are declared in the class definition, else the compiler will warn.
- Declaration and initialization are not in 2 or more different places.
- It's similar to the argument made for [Local Variables](https://google.github.io/styleguide/cppguide.html#Local_Variables) in the Google C++ Style Guide.